### ISSUE TITLE
Right Click context menu is not showing up on html5 videos/containers, when gesture key is also the right mouse button

### ISSUE THREAD

**Issue by tokariu** [marklieberman/foxygestures#272]
I'm using Firefox 62 on Windows 7 and latest FoxyGestures addon. 

I noticed an annoying problem: when the gesture key is the right mouse button, the right-click context menu on html5 videos doesn't show up anymore. Furhtermore: the first right-click does not open the context menu over the video. the SECOND right-click however opens the right-click context menu over the video. 
all other right-click context menu (just somewhere else on the page, not hovering html5 videos) work just fine with the first right-click.

When I disable foxygestures addon, the context menu of html5 videos shows up on the 1st right-click, just as it used to be. after enabling foxygestures again, I have to right-click twice to open the context menu.

Could this be fixed? It's very annoying, otherwise it's very good.

**Comment by marklieberman** [comment]
I can't seem to reproduce this. I tried a few sites but the right-click menu works normally. Maybe link me a page that isn't working for you. 

Gestures don't work at all when the video controls are visible. FF did a change a couple versions back that stopped sending mouse events when the controls are visible. Are you having this double-click issue even when video controls are visible?

**Comment by tokariu** [comment]
yes, if you right-click somewhere else on the page before you right-click on the video it works, because it counts already as the 2nd click then. after that, all right clicks work until you reload the tab or open another page with a video.

I have no idea, all the videos I on the internet just seem to have controls enabled, so whenever I hover my mousepointer over the video, the player controls show up. therefor this happens everytime I right-click over the video.

Maybe I should mention that the other gesture addon "gesturefy" has the same problem. the right click context menu does not work. and iirc, the right click menu also doesn't work the first time when you click on non-video page content, so the problem is even bigger there.

I also discovered a gesture addon called "Mausgesten 2.3" by Ternary, which is a very basic gesture addon. However, with this gesture addon enabled, the right click context-menu over videos is just working fine. Maybe there is something to discover or learn from.

**Comment by marklieberman** [comment]
I'll see what I can do to fix this, if possible. However, when I talk about the video controls I mean the ones that are displayed when you go _Right-Click > Show Controls_ (or if the video element has controls="true"). Those are the native Firefox HTML5 video controls, and they don't pass mouse events so gestures don't work on them. Most video players on websites implement their own controls that do not interfere with gestures.

**Comment by Mithgol** [comment]
Has this issue ever been reported to the developers of Firefox? — for example, is the bug https://bugzilla.mozilla.org/show_bug.cgi?id=1180031 the same or not?

**Comment by marklieberman** [comment]
@Mithgol Gestures don't work on video tags with controls=true because the native video controls do not pass mouse events to the DOM. See: https://bugzilla.mozilla.org/show_bug.cgi?id=1412617
